templates/ChoiceSchema.stencil

```diff
@@ -0,0 +1,6 @@
+
+public enum {{ choice.language.default.name }}: {{ choice.choiceType.language.default.name }}, Codable {
+{% for aChoice in choice.choices %}
+    case {{ aChoice.value | lowercase }}
```

Comment:
Remove the `| lowercase`. All naming and casing issues are the responsibility of the`SwiftNamer`. For now the casing will be messed up since it will pull from `language.default`

Reply:
Removed lowercase.